jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererTelemetryService.kt

```diff
@@ -337,12 +370,21 @@ class CodeWhispererTelemetryService {
             // the order of the following matters
             // step 1, send out current decision
             previousUserTriggerDecisionTimestamp = Instant.now()
+            val referenceCount = if (detailContexts.map { it.recommendation }.any { it.hasReferences() }) 1 else 0
```

Comment:
@panshao612, do we only care about if there is  code reference(s) or not? OR the actual count,  asking because the name itself suggests the "count" instead of "yes/no".

Reply:
@Will-ShaoHua we discussed this and aligned that this will currently be a binary value.

Reply:
ack